regolith/schemas.py

```diff
@@ -3818,11 +3897,23 @@
         "pi_id": {"description": "id of the PI",
                   "required": False,
                   "type": "string"},
-        "status": {"description": f"status of the projectum. "
-                                  f"Allowed values are {', '.join(PROJECTUM_STATUS)}",
+        "status": {"description": "status of the projectum. "
+                                  "Allowed values are 'proposed', 'started', 'converged', 'finished', 'cancelled', 'paused'",
                    "required": False,
                    "type": "string",
-                   "eallowed": PROJECTUM_STATUS}
+                   "eallowed": PROJECTUM_STATUS},
+        "other_urls": {"description": "link to remote repository. e.g. analysis or data repositories",
+                         "required": False,
+                         "type": "list"},
+        "product_url": {"description": "url for manuscript or code repository",
+                        "required": False,
+                        "type": "string"},
+        "related_projecta": {"description": "list of id's of related projecta",
```

Comment:
please remove related projecta

Reply:
Done.
Once this PR is merged I will adjust the database